ddtrace/internal/agent.py

```diff
@@ -148,11 +149,11 @@ def get_connection(url, timeout=DEFAULT_TIMEOUT):
     raise ValueError("Unsupported protocol '%s'" % parsed.scheme)
 
 
-def _healthcheck():
+def info():
```

Comment:
why `info` instead of `_info`? it's not a public method right?

Reply:
This is already an internal module, so I don't think we need to worry about this. @Kyle-Verhoog WDYT?

Reply:
That's correct @P403n1x87 https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces

Reply:
Yeah this is internal, we should be fine here :+1:

@avara1986 we explicitly make the `internal` module (and its submodules) private (see Emmett's link)